assets/js/blocks/product-categories/editor.scss

```diff
@@ -1,3 +1,13 @@
 .wc-block-product-categories.wc-block-product-categories ul {
 	margin-left: 20px;
 }
+.wc-block-product-categories {
+	&.components-placeholder {
+		// Reset the background for the placeholders.
+		background-color: rgba( 139, 139, 150, .1 );
```

Comment:
Is this declaration required? For me this background color is already inherited from `.components-placeholder`.